browser/scripts/core.js

```diff
@@ -92,9 +92,9 @@ AssetTracker.prototype.signal_failed = function()
 AssetTracker.prototype.signal_update = function()
 {
 	var l = this.listeners;
-	
-	if(E2.dom.load_spinner)
-		E2.dom.load_spinner.css('display', this.started === (this.completed + this.failed) ? 'none' : 'inherit');
+	var prc = (this.completed + this.failed) / (this.started / 100);
+	if(E2.dom.progressbar.is(':visible'))
```

Comment:
pls check for existence of E2.dom and E2.dom.progressbar, they might not be there in eg. the player

`if (E2.dom && E2.dom.progressbar && E2.dom.progressbar.is(':visible'))`